docs/exts/eliot_metrics.py

```diff
@@ -62,7 +64,7 @@ def generate_docs(self, clspath):
         table = []
         table.append(("Key", "Type"))
         for key, metric in metrics.items():
-            table.append((":py:data:`%s`" % key, metric.stat_type))
+            table.append((f":py:data:`{key}`", metric["type"]))
```

Comment:
Some of the changes in this file are to switch to f-strings. However, the other set of changes covers how the metric record was originally a tuple of `(stat_type, description)` and is now a dict with `type` and `description` keys.